README.md

```diff
@@ -2,79 +2,80 @@
 
 # AICA Package Template
 
-A software development kit for creating custom components for AICA applications.
+A software development kit for creating custom components and controllers for AICA applications.
 
 Full documentation for creating, testing and packaging custom components is available at
-https://docs.aica.tech/docs/category/custom-components.
+https://docs.aica.tech/docs/category/custom-components. A page for creating custom controller is coming soon.
 
-You will need to have Docker installed in order to develop or build your component package (see
-[Docker installation](https://docs.docker.com/get-docker/)).
+You will need to have Docker installed in order to develop or build your package (see
+[Docker installation](https://docs.docker.com/get-docker/)), and to run the initialization wizard.
 
-## Create a custom package repository
+## Create a custom package template repository
 
-To create a component package, create a new repository in GitHub using this repository as a template.
+To create a package template, create a new repository in GitHub using this repository as a template.
 
 ![Use this template menu](docs/creation-1.png) ![Create new repository menu](docs/creation-2.png)
 
 Alternatively, you can also clone this repository locally and create a new repository from it.
 
 ```bash
-git clone git@github.com:aica-technology/component-template.git my_component_package
+git clone git@github.com:aica-technology/package-template.git my_collection
 ```
 
-## Initialize the template component package
+## Running the package template wizard
 
-This template repository uses the placeholder package name `template_component_package` and includes four different
-component types: C++ component, C++ lifecycle component, Python component, and Python lifecycle component.
+When you first clone the template repository, no source will be populated. First, start the initialization wizard by
+executing `./initialize_templates.sh`. An interactive console UI will guide you through the necessary steps to populate
+the directory. During this process, you may opt in/out to specific templates or types of components to generate.
 
-The first step is to rename the package to a meaningful custom name and remove any unnecessary component types. You can
-accomplish this by using the included [`initialize_package`](./initialize_package.sh) script. In a terminal, run
+In case you already ran the wizard, but are not happy with your selection, you may re-run it as if it was the first run.
+This will remove any files previously created through the wizard.
```

Comment:
If this is a destructive operation (i.e. if the wizard also removes files that have been edited post-initialization), then I recommend making this statement bold.
```suggestion
**This will remove any files previously created through the wizard!**
```